tox.ini

```diff
@@ -87,4 +87,23 @@ setenv=
     {[testenv]setenv}
     DJANGO_SETTINGS_MODULE=cfgov.settings.test
 commands=
-    ./manage.py runscript missing_migrations 
+    ./manage.py runscript missing_migrations
+
+[testenv:validate-assets]
+# Invoke with: tox -e validate-assets
+# Ensure all assets are generated without error.
+envdir={[testenv]envdir}-validate-assets
+changedir={toxinidir}
+recreate=False
+deps=
+    -r{toxinidir}/requirements/optional-public.txt
+    -r{toxinidir}/requirements/optional-private.txt
+    {[testenv]deps}
+    mysql
```

Comment:
If you add `-r{toxinidir}/requirements/mysql.txt`, you'll be including the `mysql` package we install for local and production.

Reply:
fixed in https://github.com/cfpb/cfgov-refresh/pull/3537/commits/70d709ce9ce0ce058da1cfa9761914817d8341ca